core/src/main/java/org/testcontainers/utility/DockerImageName.java

```diff
@@ -0,0 +1,18 @@
+package org.testcontainers.utility;
+
+public class DockerImageName {
```

Comment:
there's a popular best practice to make utility classes final with private constructor